### PR TITLE
[prometheus-postgres-exporter] allow overriding image registry

### PR DESCRIPTION
**Pull request by yann-soubeyrand**
#### What this PR does / why we need it

This PR allows overriding the registry from which to fetch the Prometheus Postgres exporter, independently from the repository. This is useful in cases such as when using AWS ECR pull through cache where the registry must be changed but not the repository.

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by zeritti — CHANGES_REQUESTED** [review]
Thank you, @yann-soubeyrand, for your PR. For a major change which this indeed is, please, add a short upgrade note in README to advise users on how to adjust custom values before upgrading (applicable to users who modified the repository field).

**Comment by yann-soubeyrand** [comment]
@zeritti you’re right, I should have done this in the first place :wink:

File: charts/prometheus-postgres-exporter/README.md [hunk 1]
```diff
@@ -40,6 +40,10 @@ _See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command doc
 
 ## Upgrading
 
+### To 6.0.0
+
+Image repository has been split into two values: the new `image.registry` value and the already existing `image.repository` value. No change is required when using the default for `image.repository`.
```
Comment (zeritti) [inline charts/prometheus-postgres-exporter/README.md:45]:
```suggestion
Image repository has been split into two values: the new `image.registry` value and the already existing `image.repository` value. No change is required when using the default for `image.repository`. If you have previously modified field `image.repository`, please, set the two fields accordingly. 
```

**Review by zeritti — CHANGES_REQUESTED** [review]
Thank you, @yann-soubeyrand, for the changes. I am further suggesting a small addition.

**Review by zeritti — APPROVED** [review]

**Pull request merged**